src/utils/builder.ts

```diff
@@ -20,24 +20,38 @@ export function createNewPathway(name: string, description?: string, pathwayId?:
 }
 
 export function exportPathway(pathway: Pathway): string {
```

Comment:
Setting properties to undefined isn't really removing them. The exported pathway will still have the property there.

Reply:
When the pathway is exported, `JSON.stringify()` removes any properties with an undefined property. Because we need to create new pathways, I find this easier to manage.